code-challenges.test.js

```diff
@@ -9,37 +9,95 @@
+// // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
 
-// a) Create a test with expect statements for each of the variables provided.
+// // a) Create a test with expect statements for each of the variables provided.
 
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+describe("multBy3", () => {
+    it("takes in an array of numbers and returns an array with all the numbers multiplied by 3.", () => {
+      const numbersArray1 = []
+      const numbersArray2 = []
+      expect(multBy3(numbersArray1)).toEqual([])
+      expect(multBy3(numbersArray2)).toEqual([])
+    })
+  })
+
+
+// Pseudo Code: 
+// Create a function called multBy3
+// input a param of array 
+// return the array and use the method map() to access the indexes
+// multiply the values within the array by 3
+
 
 // b) Create the function that makes the test pass.
+const multBy3 = (array) => {
+  return array.map(value => value * 3)
+}
 
+// OUTPUT:
+// ✓ takes in an array of numbers and returns an array with all the numbers multiplied by 3. (1 ms)
 
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
+// Snapshots:   0 total
 
+// // //_________________________________________________________________________________________________________________
 // --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
-
 // a) Create a test with expect statements for each of the variables provided.
 
 const num1 = 15
-// Expected output: "15 is divisible by three"
+// // Expected output: "15 is divisible by three"
 const num2 = 0
-// Expected output: "0 is divisible by three"
+// // Expected output: "0 is divisible by three"
 const num3 = -7
-// Expected output: "-7 is not divisible by three"
-
+// // Expected output: "-7 is not divisible by three"
+
+describe("divisibleBy3", () => {
+    it("takes in a number as an argument and decides if the number is evenly divisible by 3 or not", () => {
+      const num1 = []
+      const num2 = []
+      const num3 = []
+      expect(divisibleBy3(num1))
+      expect(divisibleBy3(num2))
+      expect(divisibleBy3(num3))
+    })
+  })
+
+// Pseudo Code: 
+// create a function called "divisibleBy3"
+// create a param called "number"
+// create an if else statement
+// input the number modulo 3 strictly equal to 0
+// return divisible by 3 if true
+// return not divisible by 3 if not true
 
 // b) Create the function that makes the test pass.
+// // this is the passing test
+const divisibleBy3 = (number) => {
+    if  (number % 3 === 0){
+        return "divisble by 3"
+    } else{
+      return "not divisible by 3"
+    }
+  }
```

Comment:
Awesome!  Logic looks great!  You can make your return more dynamic by using string interpolation.